src/main/java/competition/subsystems/drive/swerve/SwerveSteeringSubsystem.java

```diff
@@ -342,5 +342,17 @@ public void periodic() {
         }
 
         currentModuleHeading.set(getCurrentValue());
+
+        motorController.periodic();
```

Comment:
I don't know how, but:
- This is the only significant difference between steering and drive parts of the module
- Only the steering fails to actuate
- The `periodic()` call does set PID values

All of this makes me suspicious.

Reply:
Yeah, this is it. When the Spark is created it has its own internal set of PID values, which aren't updated by calls to SetP, SetD, etc. When Periodic is called, on the first call, it sets all PID values from its internal set.